packages/lesswrong/lib/utils/schemaUtils.ts

```diff
@@ -41,7 +41,7 @@ const generateIdResolverMulti = ({collectionName, fieldName, getKey = (a=>a)}) =
 // If the user can't access the document, returns null. If the user can access the
 // document, return a copy of the document in which any fields the user can't access
 // have been removed. If document is null, returns null.
-export const accessFilterSingle = (currentUser, collection, document) => {
+export const accessFilterSingle = <T extends DbObject>(currentUser:DbUser|null , collection:CollectionBase<T>, document: T) : T|null => {
   const { checkAccess } = collection
```

Comment:
I approved, but then realized I had missed this section, which I guess @jimrandomh should at least quickly sanity check.